README.md

```diff
@@ -783,9 +783,21 @@ customButtons: [{
     window.location.href = 'http://www.example.com';
   }
 }]
+
+// An example that adds a custom button with a localized title underneath the login form on the primary auth page
+customButtons: [{
```

Comment:
what about this? so that developer know they must specify i18n

```
i18n: {
  en: {..}
},
customButtons: [
  {...}
]

```

Reply:
Good idea, I've added the `i18n` field in the README for this example